src/main/java/inventorysetups/InventorySetupsPersistentDataManager.java

```diff
@@ -72,15 +77,7 @@ public void loadConfig()
 		// Handles migration of old setup data
 		handleMigrationOfOldData();
 
-		Type setupTypeV2 = new TypeToken<ArrayList<InventorySetupSerializable>>()
-		{
-
-		}.getType();
-		List<InventorySetupSerializable> issList = new ArrayList<>(loadData(CONFIG_KEY_SETUPS_V2, setupTypeV2));
-		for (final InventorySetupSerializable iss : issList)
-		{
-			inventorySetups.add(InventorySetupSerializable.convertToInventorySetup(iss));
-		}
+		inventorySetups.addAll(loadV3Setups());

```

Comment:
I prefer we pull out `loadV3Setups()` into a var first, helps with debugging

Reply:
done